### DISCUSSION TITLE
Media browser shortcut to clips

### DISCUSSION THREAD

**Discussion by wtip** [blakeblackshear/frigate#1122] [Ask A Question]
Any idea how to have a more direct shortcut to view clips in home assistant?
Right name I click on Media Browser -> Frigate -> Clips. I'd like to have a 1 click shortcut from the Overview home screen.

**Comment by muzzak123** [comment]
I don't think Media Browser allows direct HREFs to sub folders, but you can add a media source in your config pointing directly to the clips folder. This will only make it 2 clicks.
```
homeassistant:
  media_dirs:
    clips: /media/frigate/clips

**Reply by blakeblackshear** [reply]
You won't get any thumbnails that way.

**Comment by bmutari** [comment]
Probably a home assistant, not a frigate problem, but I would love this feature too if anyone has a good work around.
